eden/scm/sapling/ext/github/submit.py

```diff
@@ -170,6 +170,18 @@ async def update_commits_in_stack(
         params = await create_serial_strategy_params(
             ui, partitions, github_repo, origin
         )
+
+    max_pull_requests_to_create = ui.configint("github", "max-prs-to-create", "-1")
+    if (
+        max_pull_requests_to_create >= 0
+        and params.pull_requests_to_create
+        and len(params.pull_requests_to_create) > max_pull_requests_to_create
+    ):
+        raise error.Abort(
+            _("refused to create %d pull requests, max is %d")
+            % (len(params.pull_requests_to_create), max_pull_requests_to_create)
```

Comment:
nit:
```suggestion
            _("refused to create %d pull requests, max is %d\nif you want to open %d pull requests, run again with `--config max-prs-to-create=-1")
              % (len(params.pull_requests_to_create), max_pull_requests_to_create, len(params.pull_requests_to_create) )

```

Reply:
Thanks for the suggestion!
I've updated the PR, and (following our [discussion on Discord](https://discord.com/channels/1042527964224557107/1042527965256364157/1199592604812189706)) set the default value to 5.

Still open to further feedback of course.